### ISSUE TITLE
Week 3 Review

### ISSUE THREAD

**Issue by Anniechanster** [deltatrio/web-app#13]
Completed the sketching, wireframes, visual assets, and interactive wireframes for all screen sizes.

**Comment by Anniechanster** [comment]
The sprint succeeded because everyone got things done ahead of time. Our team worked together to complete the interactive wireframe and I am satisfied with the result. 

I took the time to research on what the different beer vocabulary means before I created the icons. After creating the icons, I arranged them onto the prototype to fit the design. At first, the icons didn't seem to fit the design of the prototype because it had long descriptions. This was fixed by shortening the descriptions into abbreviations for a clean look.

**Comment by kaylaobritsch** [comment]
We succeeded in the week 3 sprint by collaborating with each other—Raph handed me the sketches, I completed the XD prototype layout and then handed it off to Annie to complete the icon designs. Our team worked really well together this week to complete the final UX design. I succeeded by taking the assets I was given to create a high fidelity interactive prototype in three screen sizes. I also took the assets and made the InVision board to help us stay on track.

It was hard this week to collaborate on the XD prototype. It was hard to take a sketch that I didn't create and prototype for it. My team members might all have different visions of the app idea and how it should look.

**Comment by refkoer** [comment]
We started our work with a lot of research, collaboration and a lot of brain storming, we did our work ahead of time. I did some industry research with similar apps and how they function UI/Ux wise, and with those information I started creating the wireframe sketches in class and handed it in to my group so we could continue on working on it in Adobe XD. We are pretty happy with the end result.

It was easy to communicate with my group and I feel like we did very well because of that.
